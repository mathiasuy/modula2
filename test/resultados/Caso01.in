ingresarCentro UY Uruguay Montevideo 100
ingresarCentro AR Argentina Cordoba 50
ingresarCentro BR Brasil Pelotas 80
ingresarCentro PY Paraguay Asuncion 30
ingresarTramo UY AR 30
ingresarTramo AR BR 80
ingresarTramo AR PY 20
ingresarTramo BR PY 50
ingresarPaquete 1 UY AR 3 [ costo ]
ingresarPaquete 2 UY BR 3 [ costo ]
ingresarPaquete 3 AR BR 3 [ costo ]
ingresarPaquete 4 UY PY 3 [ costo ]
procesarPaquete 1
procesarPaquete 3
procesarPaquete 4
removerTramo AR PY
recalcularRuta 4
procesarPaquetesCentro AR
procesarPaquetesCentro UY
listarCDPs
listarPaquetes
listarTramos
imprimirRutasOrigDest UY PY 5
imprimirRutaOpOrigDest UY BR 5
infoCDP UY
infoPaquete 4
infoTramo UY AR
infoCDPActualPaquete 4
guardar Caso01.psal
cargar Caso01.psal
salir
