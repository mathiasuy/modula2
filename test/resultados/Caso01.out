Bienvenidos a InCorreoProg2
version 1.0
InCo-FIng-UdelaR

1> El CDP fue dado de alta
2> El CDP fue dado de alta
3> El CDP fue dado de alta
4> El CDP fue dado de alta
5> El tramo fue dado de alta
6> El tramo fue dado de alta
7> El tramo fue dado de alta
8> El tramo fue dado de alta
9> El paquete fue dado de alta
10> El paquete fue dado de alta
11> El paquete fue dado de alta
12> El paquete fue dado de alta
13> El paquete fue procesado y ha llegado a su destino
14> El paquete fue procesado y ha llegado a su destino
15> El paquete fue procesado
16> El tramo fue dado de baja
17> La ruta ha sido recalculada
18> 4 - El paquete fue procesado
19> 2 - El paquete fue procesado
20> AR BR PY UY 
21> 2
 4
22> AR-BR
BR-PY
UY-AR
23> 
PY
 BR
  AR
   UY
24> 3
1
UY
AR
BR
25> UY
Uruguay
Montevideo
100
26> 4
[ costo ]
3
4
3
UY
AR
BR
PY
27> UY
AR
30
28> El paquete se encuentra en transito en el centro BR
29> Los datos han sido guardados
30> Los datos han sido cargados
31> Fin!